change/@fluentui-react-f721ac8e-1f5f-4966-87d0-2b1a6d6f3ba5.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Add onClick logic to container div",
```

We should have a better changelog comment.

```suggestion
  "comment": "Add onClick logic to container div so that it focuses back on item when you hit the item limit.",
```